ublox_gps/src/gps.cpp

```diff
@@ -392,21 +392,26 @@ bool Gps::configUsb(uint16_t tx_ready,
   return configure(port);
 }
 
-bool Gps::configRate(uint16_t meas_rate, uint16_t nav_rate) {
+bool Gps::configRate(uint16_t meas_rate, uint16_t nav_rate, bool wait) {
   RCLCPP_DEBUG(logger_, "Configuring measurement rate to %u ms and nav rate to %u cycles",
                meas_rate, nav_rate);
 
   ublox_msgs::msg::CfgRATE rate;
   rate.meas_rate = meas_rate;
   rate.nav_rate = nav_rate;  //  must be fixed at 1 for ublox 5 and 6
   rate.time_ref = ublox_msgs::msg::CfgRATE::TIME_REF_GPS;
-  return configure(rate);
+  return configure(rate, wait);
 }
 
-bool Gps::configRtcm(const std::vector<Rtcm> & rtcms) {
+bool Gps::configRtcm(const std::vector<Rtcm> & rtcms, bool wait) {
   for (const Rtcm & rtcm : rtcms) {
     RCLCPP_DEBUG(logger_, "Setting RTCM %d Rate %u", rtcm.id, rtcm.rate);
-    if (!setRate(ublox_msgs::Class::RTCM, rtcm.id, rtcm.rate)) {
+
+    ublox_msgs::msg::CfgMSGS msgs;
+    msgs.msg_class = ublox_msgs::Class::RTCM;
+    msgs.msg_id = rtcm.id;
+    msgs.rates[ublox_msgs::msg::CfgMSGS::PORT_ID_UART1] = rtcm.rate;
```

Comment:
One thing to note here: I'm only setting the rate on the UART1 port.  There seem to be a lot of assumptions around this in the code, so I wasn't sure how to pass in *which* ports to set.  Happy to make some modifications around this as necessary.